torch_xla/csrc/ops/normal.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <c10/util/Optional.h>
+
+#include <vector>
+
+#include "torch_xla/csrc/ir.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+
+class Normal : public Node {
+ public:
+  Normal(const Value& mean, const Value& std, xla::uint64 seed);
+
+  std::string ToString() const override;
+
+  NodePtr Clone(OpList operands) const override;
+
+  XlaOpVector Lower(LoweringContext* loctx) const override;
+
```

Comment:
I think we should expose the seed.
I know I did not in the rrelu op, but we can change that too.